Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PaalmanPingsMonteCarloAbsorption.py

```diff
@@ -22,6 +22,9 @@ class PaalmanPingsMonteCarloAbsorption(DataProcessorAlgorithm):
     _shape = None
     _height = None
     _isis_instrument = None
+    _is_override_sample = None
```

Comment:
It took me some time to understand how exactly these flags work, can we maybe name them better? Something like `override_shape`, `override_sample_material`, `has_container`? What about overriding the container material? Is there a use case for that?

Reply:
My feeling is that it's more likely the user would want to override the sample material than the container material. The user might have multiple samples of the same shape but with different material. To support this they would load the sample shape into the workspace but leave the material blank and override it in the algorithm.

It doesn't seem as likely to me that they'd have multiple containers of the same shape and different material? But maybe I'm wrong about that...

Reply:
By overriding the container, do you mean overriding the container only without affecting the sample geometry? If so I'm afraid I'm not sure if there's a use case for this, I'd be open to suggestions